safe_transaction_service/history/models.py

```diff
@@ -1526,33 +1526,33 @@ class Meta:
     def __str__(self):
         return f"{self.safe} - {self.nonce} - {self.safe_tx_hash}"
 
-    def to_log(self, message: str) -> str:
+    def to_dict(self) -> dict:
         """
-        :param message:
-        :return: MultisigTransaction ready to be printed in a log line
+        :return: MultisigTransaction as dict
         """
         safe_tx_hash_str = to_0x_hex_str(HexBytes(self.safe_tx_hash))
-        return (
-            f"[MultisigTransaction {safe_tx_hash_str}] {message}. "
-            f"safe_tx_hash={safe_tx_hash_str} "
-            f"safe={self.safe} "
-            f"proposer={self.proposer} "
-            f"proposed_by_delegate={self.proposed_by_delegate} "
-            f"to={self.to} "
-            f"value={self.value} "
-            f"data={to_0x_hex_str(HexBytes(self.data)) if self.data else None} "
-            f"operation={self.operation} "
-            f"safe_tx_gas={self.safe_tx_gas} "
-            f"base_gas={self.base_gas} "
-            f"gas_price={self.gas_price} "
-            f"gas_token={self.gas_token} "
-            f"refund_receiver={self.refund_receiver} "
-            f"signatures={to_0x_hex_str(HexBytes(self.signatures)) if self.signatures else None} "
-            f"nonce={self.nonce} "
-            f"failed={self.failed} "
-            f"origin={self.origin} "
-            f"trusted={self.trusted} "
-        )
+        return {
```

Comment:
Maybe we can use any of these alternatives to return a dictionary? https://stackoverflow.com/a/29088221

`model_to_dict` sound like a good option